configs/torch-2.9/torch_variant_config.toml

```diff
@@ -121,27 +121,27 @@ deps_add_list    = [
     "nvidia-nvtx==13.0.39; platform_system == 'Linux' and variant_label == 'cuda13.0'",
 
     # XPU
-    "dpcpp-cpp-rt==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
+    "dpcpp-cpp-rt==2025.2.1; 'intel' in variant_namespaces",
     "impi-rt==2021.16.1; platform_system == 'Linux' and platform_machine == 'x86_64' and 'intel' in variant_namespaces",
-    "intel-cmplr-lib-rt==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "intel-cmplr-lib-ur==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "intel-cmplr-lic-rt==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "intel-opencl-rt==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "intel-openmp==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "intel-pti==0.13.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "intel-sycl-rt==2025.2.1; platform_system == 'Linux' and 'intel' in variant_namespaces",
-    "mkl==2025.2.0; platform_system == 'Linux' and 'intel' in variant_namespaces",
+    "intel-cmplr-lib-rt==2025.2.1; 'intel' in variant_namespaces",
```

Comment:
For reviewers:

Limiting the following 3 package to `platform_system == 'Linux' and platform_machine == 'x86_64'` is not a mistake as these packages are needed for XCCL distributed backend and it's not available for Linux:
* `impi-rt`
* `oneccl-devel`
* `oneccl`